### ISSUE TITLE
Model training fails  with error in R

### ISSUE THREAD

**Issue by ThomasWolf0701** [catboost/catboost#1772]
Problem:
Catboost in R does not work with  'RMSEWithUncertainty',.
I did not even call any feature importance function. Just ftrained the model:

library(catboost)

train_pool_path <- system.file("extdata", "adult_train.1000", package = "catboost")
test_pool_path <- system.file("extdata", "adult_test.1000", package = "catboost")
cd_path <- system.file("extdata", "adult.cd", package = "catboost")
train_pool <- catboost.load_pool(train_pool_path, column_description = cd_path)
test_pool <- catboost.load_pool(test_pool_path, column_description = cd_path)

fit_params <- list(
  iterations = 100,
  loss_function = 'RMSEWithUncertainty',
  #ignored_features = c(4, 9),
  border_count = 32,
  depth = 5,
  learning_rate = 0.03,
  l2_leaf_reg = 3.5,
  train_dir = 'train_dir')
model <- catboost.train(train_pool,params =  fit_params)

Error in catboost.get_feature_importance(model, learn_pool) : 
  C:/Program Files (x86)/Go Agent/pipelines/BuildMaster/catboost.git/catboost/private/libs/target/data_providers.cpp:391: Non-Multiclassification and Non-Multiregression compatible metric (RMSEWithUncertainty) specified for a multidimensional model

catboost version: 0.26
Operating System: windows
CPU:Intel
# GPU:

**Comment by LyzhinIvan** [comment]
Fixed by 5de4439.

**Comment by kizill** [comment]
Will release fix in 0.26.1
